### DISCUSSION TITLE
Is it possible to change an entry section to mandatory on Sheltermanager?

### DISCUSSION THREAD

**Discussion by ADIN2014** [sheltermanager/asm3#1462] [Q&A / Help]
Unfortunately, people keep omitting entering the original owner details for hand ins on Sheltermanager.  Is there a way to edit the section to make it mandatory when creating a new animal on the system?

**Comment by Tetzor** [comment]
I'm afraid not, as there won't always be original owner information, it isn't possible to make the field mandatory.
